src/js/stores/document.js

```diff
@@ -907,7 +907,8 @@ define(function (require, exports, module) {
             var documentID = payload.documentID,
                 layerIDs = payload.layerIDs,
                 document = this._openDocuments[documentID],
-                nextLayers = document.layers.setCharacterStyleProperties(layerIDs, { postScriptName: postScriptName }),
+                tmpLayers = document.layers.setCharacterStyleProperties(layerIDs, { postScriptName: postScriptName }),
+                nextLayers = tmpLayers.setTextWarningLevel(layerIDs, 0),
```

Comment:
Also, are you sure that this is the _only_ property change that will cause a re-layout? What about if you change the size, tracking, leading or alignment?